.bazelrc

```diff
@@ -3,6 +3,8 @@
 # Settings that apply only to CI are in .github/workflows/ci.bazelrc
 
 common --enable_bzlmod
+# PR 997 not merged yet
```

Comment:
Can we put a link?

Reply:
it will merge in a couple mins, will remove before merging this